src/components/Category/CategoryList.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "@/components/ui/button";
+
+import { Link } from "@/i18n/routing";
+
+const categories = [
+  { name: "DESSERTS 🍰", href: "/desserts" },
+  { name: "MAIN-DISHES 🍽️", href: "/main-dishes" },
+  { name: "APPETIZERS 🧀", href: "/appetizers" },
+  { name: "SALADS 🥗", href: "/salads" },
+  { name: "SOUPS 🍲", href: "/soups" },
+  { name: "BREAKFAST 🍳", href: "/breakfast" },
+  { name: "BEVERAGES 🧃", href: "/beverages" },
+  { name: "SNACKS 🍿", href: "/snacks" },
+  { name: "SAUCES 🍯", href: "/sauces" },
+  { name: "BREADS 🥖", href: "/breads" },
+  { name: "SEAFOOD 🦞", href: "/seafood" },
+  { name: "VEGETARIAN 🥦", href: "/vegetarian" },
+  { name: "GRILLED 🍖", href: "/grilled" },
+  { name: "HOLIDAY-SPECIALS 🎄", href: "/holiday-specials" },
+  { name: "KIDS-MENU 🍔", href: "/kids-menu" },
+];
+//TODO: Async, get from DB
+const CategoryList = ({ query }: { query: string }) => {
+  const filteredCategories = Array.isArray(categories)
```

Comment:
oh i wanted to say that we can use `useSearchParams `from next but then we would have to change that component to client.